plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1053,6 +1054,12 @@ public void testInsertIntoPartitionedNonLowercaseColumnTable()
                         "(null, null, null, null, 8.0, null, null)");
     }
 
+    @Test
```

Comment:
We already have `TestDeltaLakeColumnMappingMode.testColumnMappingMode`. Could you update the test instead?

Reply:
Thank you for the review, that is indeed a much better location for the test. I've updated it there and removed the previous smoke test.